sym_api_client_python/datafeed_event_service.py

```diff
@@ -445,12 +445,24 @@ async def promoted_to_owner(self, payload):
         for listener in self.room_listeners:
             await listener.on_room_member_promoted_to_owner(promoted_to_owner_data)
 
-    async def demoted_to_owner(self, payload):
-        log.debug('async demoted_to_Owner')
+    async def demoted_from_owner(self, payload):
```

Comment:
Are we breaking the API here?

Reply:
Not really. The `AsyncDatafeedEventService` is meant to be accessed from `SymBotClient`. These methods are called  by the `handle_events` method which references all the methods for each event type: https://github.com/SymphonyPlatformSolutions/symphony-api-client-python/blob/master/sym_api_client_python/services/abstract_datafeed_event_service.py#L37
Bottom line: this was not meant to be called by a developer as part of a public API.